keras_hub/src/models/efficientnet/convbnact.py

```diff
@@ -0,0 +1,136 @@
+import keras
```

Comment:
rename file to be more readable - convolution_batch_norm_activation.py

Reply:
this conflicts w/ the below review, so I chose the later review.